Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.11-alpine3.8 as builder
+FROM golang:1-alpine as builder
```

Comment:
Why did you omit the exact version here?

Reply:
just to use the latest version of the golang